src/modules/visualBuilder/pageBuilder.js

```diff
@@ -0,0 +1,146 @@
+export const initialiseCTBuilder = (url, variant, details) => {
+  document.addEventListener('DOMContentLoaded', () => onContentLoad(url, variant, details))
+}
+
+let container
+
+function onContentLoad (url, variant, details) {
+  document.body.innerHTML = ''
+  container = document.createElement('div')
+  container.id = 'overlayDiv'
+  container.style.position = 'relative' // Ensure relative positioning for absolute positioning of form
+  container.style.display = 'flex'
+  document.body.appendChild(container)
+  const overlayPath = 'https://d2r1yp2w7bby2u.cloudfront.net/js/lib-overlay/overlay.js'
```

Comment:
lets remove this hard coded string here. Add a new constants file inside `visualBuilder` and add this string

Reply:
This is Done